sbi/neural_nets/net_builders/vector_field_nets.py

```diff
@@ -751,6 +849,97 @@ def forward(self, x: Tensor, cond: Tensor) -> Tensor:
         return x
 
 
+class MaskedDiTBlock(nn.Module):
```

Comment:
Why do you need to redefine this? Couldnt you add mask as optional keyword argument to the existing implementation.